src/hevm/test/contracts/pass/dsProvePass.sol

```diff
@@ -0,0 +1,90 @@
+import "ds-test/test.sol";
+import "lib/erc20.sol";
+
+contract ConstructorArg {
+    address immutable public a;
+    constructor(address _a) public {
+        a = _a;
+    }
+}
+
+contract SolidityTest is DSTest {
+    ERC20 token;
+
+    function setUp() public {
+        token = new ERC20("Token", "TKN", 18);
+    }
+
+    function prove_trivial() public {
+        assertTrue(true);
+    }
+
+    function prove_easy(uint v) public {
+        if (v != 100) return;
+        assertEq(v, 100);
+    }
+
+    function prove_add(uint x, uint y) public {
+        unchecked {
+            if (x + y < x) return; // no overflow
+            assertTrue(x + y >= x);
+        }
+    }
+
+    //function prove_balance(address usr, uint amt) public {
```

Comment:
Were these committed commented-out by accident? It's useful to comment out during debugging, so I'm thinking this may have been accidental?

Reply:
Ah, I see. Sorry, I was afraid this was unintentional.